pkg/api/const.go

```diff
@@ -223,7 +223,7 @@ const (
 	// https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#load-balancer.
 	DefaultMaximumLoadBalancerRuleCount = 250
 	// DefaultEnableAutomaticUpdates determines the aks-engine provided default for enabling automatic updates
-	DefaultEnableAutomaticUpdates = true
+	DefaultEnableAutomaticUpdates = false
```

Comment:
For some reason I thought this already defaulted to false, so lgtm

PS this was being unit tested :)

Reply:
ooo - i'll update

Reply:
One funny thing is that we disable windows updates in the aks-engine VHDs we produce in the windows registry. I thought this was defaulting to false too.

Reply:
Ah, that's why I thought we have been doing this for a long time. I still think this change is good for folks who choose not to use the VHD.